pkgs/development/compilers/gcc/8/default.nix

```diff
@@ -130,7 +131,12 @@ stdenv.mkDerivation ({
 
   builder = ../builder.sh;
 
-  src = fetchurl {
+  src = if stdenv.targetPlatform.isVc4 then fetchFromGitHub {
+    owner = "itszor";
+    repo = "gcc-vc4";
```

Comment:
GCC forks are hard to maintain. Not sure how to handle this though

Reply:
This also looks like it's actually GCC6, not GCC8. This would be a good opportunity figure out if we can make it possible to select Compilers when cross compiling. Right now we just do this:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/cross/default.nix#L54-L62

Adding some option for what GCC version you need would be useful.

/cc @Ericson2314

Reply:
Yeah let's the diff to be gcc 6 too. We can figure out compiler selection and stuff later. The source replacement isn't fun, but it's the least invasive option. We can bring this stuff up for the platform support RFCs.